api/helpers/game-states/moves/resolve/six.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
+  friendlyName: 'Resolve Six One-Off',
+
+  description: 'Returns new GameState resulting from resolving an six, which Scrap all Royals and Glasses Eights on the field.',
+
+  inputs: {
+    currentState: {
+      type: 'ref',
+      description: 'The latest GameState before the six scraps all royals and glasses eights',
+      required: true,
+    },
+  },
+  sync: true, // synchronous helper
+  fn: ({ currentState }, exits) => {
+    let result = _.cloneDeep(currentState);
```

Comment:
Since we're not reassigning to result we can use const
```suggestion
    const result = _.cloneDeep(currentState);
```